EventStorming.md

```diff
@@ -0,0 +1,16 @@
+# #EventStorming
+
+The [#EventStorming](https://www.eventstorming.com/) sessions resulted 
+in the identification of two bounded contexts.
+
+- **Sessions** manages user sessions and handles authentication via GitHub. 
+  It is also responsible for automatically refreshing the session if it has expired.
+
+<img src="./img/event-storming/sessions-bc.jpg" width="700px" alt="Sessions bounded context">
+
+- **Mentions** manages user mentions and retrieves new mentions from GitHub. 
```

Comment:
- **Mentions**, meticulously following all cases of someone mentioning end-users of Pingh at GitHub.